flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/utils/CloneUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.kubernetes.operator.utils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/** Utility class for cloning objects. */
+public class CloneUtils {
```

Comment:
this is duplicate with `org.apache.flink.kubernetes.operator.admission.admissioncontroller.clone.ObjectMapperCloner` Can extract and use the common one

Reply:
I'm hesitating adding the webhook module as a dependency for now. We can think of a shared core module later. Nice catch tho.

Reply:
The webhook is depend on the operator module now, we can let the webhook to use the util in the operator

Reply:
Yep, we can do this, preferably in a separate ticket.

Reply:
I agree with @Aitozi , lets reuse the clone utility method in the webhook that we just created. Should be a super simple change, let's include it in this one